pkg/rdkafka/RdKafkaContext.php

```diff
@@ -180,6 +179,58 @@ public static function getLibrdKafkaVersion(): string
         return "$major.$minor.$patch";
     }
 
+    /**
+     * @return void
+     *              JsonSerializer should be the default fallback if no serializer is specified
+     */
```

Comment:
No need to re-specify the return type, since it's already present in the PHP declaration.

As for the description, I'm not sure it's worthwile - but nothing against it either.

```suggestion
    /**
     * JsonSerializer is the default fallback, if no serializer is specified.
     */
```